app/src/main/res/menu/menu.xml

```diff
@@ -17,20 +19,20 @@
             android:visible="false"
             app:showAsAction="ifRoom"/>
 
-    <item
-            android:id="@+id/hate"
-            android:icon="@drawable/hate"
-            android:orderInCategory="100"
-            android:title="@string/hate"
-            android:visible="false"
-            app:showAsAction="ifRoom"/>
+    <!--<item-->
```

Comment:
this comment that is not useful for the project.

Reply:
solved